custom_components/places/basic_options.py

```
@@ -214,108 +223,112 @@ def add_type_or_category(
+            not coordinator.is_attr_blank("place_category")
+            and coordinator.get_attr_safe_str("place_category").lower() != "highway"
         ):
-            formatted_place_array.append(sensor.get_attr_safe_str("place_category").title().strip())
+            formatted_place_array.append(
+                coordinator.get_attr_safe_str("place_category").title().strip()
+            )
 
     def add_street_info(
         self,
         formatted_place_array: list[str],
         internal_attr: MutableMapping[str, Any],
```

Ruff (0.15.20)

[warning] Unused method argument: `internal_attr`

(ARG002)